unturned/OpenMod.Unturned/Vehicles/Events/UnturnedVehicleUnlockingEvent.cs

```diff
@@ -0,0 +1,21 @@
+﻿using OpenMod.API.Eventing;
+using OpenMod.Unturned.Players;
+using Steamworks;
+
+namespace OpenMod.Unturned.Vehicles.Events
+{
+    public class UnturnedVehicleUnlockingEvent : UnturnedVehicleEvent, ICancellableEvent
```

Comment:
Merge `UnturnedVehicleLockingEvent` and `UnturnedVehicleUnlockingEvent` into one event with a property of the locked state.